pubspec.yaml

```diff
@@ -1,24 +1,27 @@
 name: vector_map_tiles
 description: A plugin for `flutter_map` that enables the use of vector tiles.
-version: 4.0.0
+version: 5.0.0
 homepage: https://github.com/greensopinion/flutter-vector-map-tiles
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
-  flutter: ">=2.14.0"
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  flutter_map: ^4.0.0
+  flutter_map:
+    ^5.0.0
     #path: ../flutter_map
-  http: ^0.13.3
-  latlong2: ^0.8.0
+  http: ^1.0.0
+  latlong2: ^0.9.0
   path_provider: ^2.0.2
-  vector_tile_renderer: ^4.0.0
+  vector_tile_renderer:
+    ^4.0.0
     #path: ../vector_tile_renderer
   async: ^2.8.2
-  executor_lib: ^1.1.0
+  executor_lib:
+    ^1.1.0
```

Comment:
same?

Reply:
I'm not really sure tbh. I'll see if I can tidy it up later today.

Reply:
Resolved in da3eb05b3e64bbd40080feb9544dffba67d76b80.
This occurred because you also had a commented out path dependency, and the auto formatter after you run 'pub upgrade' does this. I've removed the comment to put it all back on one line.